cmd/root.go

```diff
@@ -29,9 +24,10 @@ var (
 	APIVersionV1 = "v1"
 	//dryrun          bool
 	verbose         bool
+	verboseHttp     bool
 	klogInitialized = false
 	KabURLKey       = "KABURL"
-	KabURLContext   = "KabCollections-1.0-SNAPSHOT"
+	//KabURLContext   = "KabCollections-1.0-SNAPSHOT"
```

Comment:
I think we can just get rid of this. I believe we stopped using this a while ago.